### PR TITLE
deps,src,test: fixing lint errors

### PR DESCRIPTION
**Pull request by MSLaguana**
##### Checklist

- [x] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
- [x] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)

##### Affected core subsystem(s)

deps, src, test

### CODE REVIEW

**Review by sethbrenith — APPROVED** [review]

**Comment by kfarnung** [comment]
Nice!  I didn't see this one before creating my PR, I'll rebase my changes on this afterwards.

File: src/node_api_jsrt.cc [hunk 1]
```diff
@@ -2119,15 +2119,16 @@ napi_status napi_open_callback_scope(napi_env env,
   node::async_context* node_async_context =
       reinterpret_cast<node::async_context*>(async_context_handle);
 
-  // TODO: It'd be nice if we could remove the dependency on v8::*
+  // TODO(jithomso): It'd be nice if we could remove the dependency on v8::*
```
Comment (kfarnung) [inline src/node_api_jsrt.cc]: I'd use your GitHub ID

File: src/node_api_jsrt.cc [hunk 2]
```diff
@@ -2119,15 +2119,16 @@ napi_status napi_open_callback_scope(napi_env env,
...
   // from here by changing node::CallbackScope's signature
   v8::Local<v8::Object> resource =
     v8impl::V8LocalValueFromJsValue(resource_object).As<v8::Object>();
 
   *result = reinterpret_cast<napi_callback_scope>(
-      new node::InternalCallbackScope(node::Environment::GetCurrent(env->isolate),
-                              resource,
-                              *node_async_context));
+      new node::InternalCallbackScope(
```
Comment (kfarnung) [inline src/node_api_jsrt.cc]: Generally we use two indents for continuations from the previous line.

File: test/cctest/node_test_fixture.h [hunk 3]
```diff
@@ -103,7 +103,8 @@ class EnvironmentTestFixture : public NodeTestFixture {
     Env(const v8::HandleScope& handle_scope, const Argv& argv) {
       auto isolate = handle_scope.GetIsolate();
 #if ENABLE_TTD_NODE
-      context_ = node::NewContext(isolate, false); // TODO: should we support TTD in cctest?
+      context_ = node::NewContext(isolate, false);
+      // TODO(jithomso): should we support TTD in cctest?
```
Comment (kfarnung) [inline test/cctest/node_test_fixture.h]: nit:  I'd move the comment above the line, also use your GitHub ID

**Review by kfarnung — APPROVED** [review]
LGTM with minor fixes

**Comment by MSLaguana** [comment]
Done. Waiting on the node pump to complete before I finish this PR.

**Comment by MSLaguana** [comment]
Landed in c057e77f60ab1cee08ad60345f937c456d87fd70

**Pull request closed without merging**